src/hooks/useHasHiddenAllTokens.ts

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from 'react'
+import { useVisibleBalances } from './useVisibleBalances'
+import useHiddenTokens from './useHiddenTokens'
+
+const useHasHiddenAllTokens = (): boolean => {
+  const { balances, loading } = useVisibleBalances()
+  const hiddenTokens = useHiddenTokens()
+
+  return useMemo(() => {
+    return !loading && hiddenTokens.length > 0 && hiddenTokens.length >= balances.items.length
```

Comment:
This condition is technically not correct.
`useHiddenTokens` returns all hidden tokens per chain. It does not check if those are also part of the current Safe's balance. 

`hiddenTokens.length >= balances.items.length` therefore does not really mean anything because there could for instance be 10 hidden spam tokens and 3 legit tokens in the current balance leading to 10 >= 3 but at the same time not all tokens are hidden.

Reply:
I've altered the PR to only disable the hiding of native tokens. I think it's a cleaner solution.